### PR TITLE
fix!: synth.py clean for multiple versions

### PR DESCRIPTION
**Pull request by summer-ji-eng**
Notice that `src/v1beta1/index.ts` and `src/v1/index.ts' is still in the exclude list

### CODE REVIEW

File: system-test/fixtures/sample/src/index.js [hunk 1]
```diff
@@ -20,8 +20,9 @@
 const containeranalysis = require('@google-cloud/containeranalysis');
 
 function main() {
-  const containerAnalysisClient = new containeranalysis.ContainerAnalysisClient();
+  const containerAnalysisV1Beta1Client = new containeranalysis.ContainerAnalysisV1Beta1Client();
```
Comment (alexander-fenster) [inline system-test/fixtures/sample/src/index.js:23]: This is unexpected...

Comment (alexander-fenster) [reply system-test/fixtures/sample/src/index.js:23]: The problem with this API is that clients have different names in `v1` and `v1beta1`. I think I will just change the order of versions to change this back.

File: src/index.ts [hunk 2]
```diff
@@ -12,17 +12,16 @@
...
 import * as v1beta1 from './v1beta1';
+import * as v1 from './v1';
+
 const ContainerAnalysisClient = v1.ContainerAnalysisClient;
-const GrafeasClient = v1beta1.GrafeasV1Beta1Client;
-export {v1, v1beta1, ContainerAnalysisClient, GrafeasClient};
-// For compatibility with JavaScript libraries we need to provide this default export:
-// tslint:disable-next-line no-default-export
-export default {v1, v1beta1, ContainerAnalysisClient, GrafeasClient};
+
```
Comment (xiaozhenliu-gg5) [inline src/index.ts:23]:
```
const GrafeasClient = v1beta1.GrafeasV1Beta1Client;
```
This is a breaking change, not sure if we still want to expose this client to users. For my understanding containerAnalyisis only `reroute` to use this interface and grafeas should be not visible to users. WDYT? @alexander-fenster

Comment (alexander-fenster) [reply src/index.ts:23]: Oh, I totally missed it. We don't want to make a breaking change here, let's put it back please (maybe with a synth hack). @summer-ji-eng 

I'll remove `automerge` for now, thank you @xiaozhenliu-gg5 for noticing it!

Comment (summer-ji-eng) [reply src/index.ts:23]: Thanks for reminding.

I will hold this PR for later and count as a category that export beta client library, like video-intelligence. And discuss with team how to deal with these libraries.

**Comment by JustinBeckwith** [comment]
Folks, what's up with this one?

**Comment by alexander-fenster** [comment]
@JustinBeckwith This is a pending breaking change - @summer-ji-eng do you know what is the API team position here?

**Comment by JustinBeckwith** [comment]
Honestly it's fine.

**Comment by summer-ji-eng** [comment]
duplicated PR with #214

**Pull request closed without merging**